generic/tests/cfg/pktgen_perf.cfg

```diff
@@ -13,8 +13,10 @@
     login_timeout = 20
     kvm_ver_chk_cmd = "rpm -qa qemu-kvm-rhev && rpm -qa qemu-kvm"
     guest_ver_cmd = "uname -r"
-    record_list = "pktgen_script pkt_size run_threads burst mpps"
+    record_list = "pkt_size run_threads burst mpps"
     pkt_size = 64
     numa_node = 1
     burst = 1
     pktgen_script = "pktgen_perf pktgen_sample01_simple pktgen_sample03_burst_single_flow pktgen_sample05_flow_per_thread"
+    format_base = 12
+    format_fbase = 2
```

Comment:
You have default value in the script now so not really necessary here. It could be defined later when you need to update it once again.

Reply:
I have updated.